svc-systems/chassis/get_collection_test.go

```diff
@@ -1,18 +1,16 @@
-/*
- * Copyright (c) 2020 Intel Corporation
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
+//(C) Copyright [2020] Hewlett Packard Enterprise Development LP
+//
```

Comment:
Don't remove the existing license add a new header  refer this file https://github.com/ODIM-Project/ODIM/blob/development/svc-api/rpc/chassis.go#L1